src/snowflake/connector/pandas_tools.py

```diff
@@ -204,12 +206,40 @@ def write_pandas(
     )
 
 
+def make_pd_writer(**kwargs) -> Callable:
+    """This returns a pd_writer with the desired arguments.
+
+        Example usage:
+            import pandas as pd
+            from snowflake.connector.pandas_tools import make_pd_writer
+
+            sf_connector_version_df = pd.DataFrame([('snowflake-connector-python', '1.0')], columns=['NAME', 'NEWEST_VERSION'])
+            sf_connector_version_df.to_sql('driver_versions', engine, index=False, method=make_pd_writer(quote_identifiers=False))
+
+            # to keep quote_identifiers=True
+            from functools import partial
+            sf_connector_version_df.to_sql(
+                'driver_versions', engine, index=False, method=make_pd_writer()))
+
+    Args:
+        kwargs: A dictionary that specifies which arguments to pass to pd_writer
+    """
+    # default argument map
+    default_args = {"quote_identifiers": True}
+    # overwrite with value in kwargs, if it exists
+    for k in default_args.keys():
+        if k in kwargs:
+            default_args[k] = kwargs[k]
```

Comment:
Interesting, I wasn't aware of the `update` method. I did consider doing it this way, but the only issue is that 
this "merges" the `dict`s in a way. For example, 
```python
default_args={'a': 1, 'c': 3}
kwargs={'a':2, 'b':2}
default_args.update(kwargs)
{'a':2, 'b':2,'c':3}
```
The only (minor) issue with this is we respect any new key/values in the dict (e.g. ('b', 2)) if we "merge" in this way. It shouldn't be a problem as far as correctness goes (because we won't use the new key/values) but I preferred a solution that kind of "updates but not merge" the two dicts. Let me know what you think.

Reply:
Resolved: not proceeding with `kwargs`